src/eos-updater-util.h

```diff
@@ -37,21 +37,29 @@
 G_BEGIN_DECLS
 
 #define EOS_UPDATER_ERROR (eos_updater_error_quark())
-GQuark eos_updater_error_quark (void);
+GQuark
```

Comment:
I had an impression that the types were always in a separate line in gobject, but I see I was wrong. Will revert it.